### PR TITLE
dbt and Git Extensions Public Beta [DOC-862]

### PR DESCRIPTION
**Pull request by pwseg**
### Proposed changes

- Early docs for the Extensions public beta, which includes docs for Git and dbt Model syncs.

### Merge timing

- ASAP once approved.

### CODE REVIEW

File: src/segment-app/extensions/dbt.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: dbt Extension
+---
+
+Segment's dbt extension lets you use [Reverse ETL](/docs/connections/reverse-etl/) with your existing dbt labs models and syncs to help centralize model management and versioning, reduce redundancies, and run CI checks to prevent breaking changes.
```
Comment (cmastr) [inline src/segment-app/extensions/dbt.md:5]: is 'allows you' or 'lets you' our standard?

File: src/segment-app/extensions/dbt.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+> info ""
+> Extensions, including dbt sync, is currently in public beta and is governed by Segment's [First Access and Beta Preview Terms](https://www.twilio.com/en-us/legal/tos){:target="_blank"}.
+
+## Before you begin
+
+Keep the following in mind as you set up the dbt extension:
+
+- The extension supports [dbt Core v1.7](https://docs.getdbt.com/docs/dbt-versions/core-upgrade/upgrading-to-v1.7){:target="_blank"}.
+- You can use Snowflake, Databricks, RedShift, Postgres, and BigQuery as Reverse ETL sources.
```
Comment (cmastr) [inline src/segment-app/extensions/dbt.md:23]: link these to their source topic?

File: src/segment-app/extensions/dbt.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- dbt models aren't synchronized from the dbt cloud. The model sync connects to a Git repository that loads models into Segment for use with Reverse ETL. 
+
+## Set up Git dbt Models and dbt Cloud
+
+To set up the dbt extension, you'll need:
+
+- an existing dbt account with a Git repository
+- (for job syncs:) dbt cloud with jobs already created
+
+### Git repository and dbt Models setup
```
Comment (cmastr) [inline src/segment-app/extensions/dbt.md:33]: passive voice?

File: src/segment-app/extensions/dbt.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+Follow these steps to connect the Git repository that stores your dbt Models:
+
+1. In your Segment workspace, navigate to **Settings > Extensions**.
+2. Click **Set up Git sync**.
+3. On the **Configure service credentials** page, select a service and protocol, add your SSH private key or GitHub token, then click **Next**.
+4. In the **Connect source** window, select a Reverse ETL warehouse source from the dropdown, then click **Save**.
+
+After you've saved your setup, you can configure your Git repository's settings to your needs by changing the repository, branch, dbt version, default schema, and project path.
+
+### dbt Cloud setup
```
Comment (cmastr) [inline src/segment-app/extensions/dbt.md:44]: passive voice

File: src/segment-app/extensions/dbt.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+You can also use dbt Cloud to schedule Reverse ETL syncs after a dbt Cloud job successfully runs. 
+
+To set up dbt Cloud:
+
+1. In your Segment workspace, navigate to **Settings > Extensions**.
+2. Click **Manage dbt Cloud**.
+3. Add your dbt Cloud API key, and, optionally, a custom subdomain. Click **Save**.
+
+### Model syncs
```
Comment (cmastr) [inline src/segment-app/extensions/dbt.md:54]: passive voice

**Review by cmastr — APPROVED** [review]

File: src/segment-app/extensions/dbt.md (hunk 1 above)
Comment (pwseg) [reply src/segment-app/extensions/dbt.md:5]: @cmastr We don't really have any standardized approach to this.

File: src/segment-app/extensions/dbt.md (hunk 2 above)
Comment (pwseg) [reply src/segment-app/extensions/dbt.md]: Good catch, done.

**Pull request merged**